log_analytics_workspace: support for the `daily_quota_gb` property (#8861)
terraform-providers/terraform-provider-azurerm: website/docs/r/log_analytics_workspace.html.markdown

* `retention_in_days` - (Optional) The workspace data retention in days. Possible values are either 7 (Free Tier only) or range between 30 and 730.

> * `daily_quota_gb` - (Optional) The workspace of the volume cap in gb.  Defaults to -1 (unlimited).

Reviewer: Can we adjust the statement a bit?

```suggestion
* `daily_quota_gb` - The workspace daily quota for ingestion in GB.  Defaults to -1 (unlimited).
```

Author: Do we not want the `(Optional)` in the resource?

Reviewer 2: we do
